### PR TITLE
 BPK-2414 Support corner radius on button

### PR DESCRIPTION
**Pull request by amolgupta**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:29bfda01fb18d1ea3320325f240aa4ba3b143143`

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:93f0824b2d1c28f86b0e626b0311b9b479a29429`

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:589af1db632b774ad84c8a77d313f67581341239`

File: Backpack/src/main/java/net/skyscanner/backpack/button/BpkButton.kt [hunk 1]
```diff
@@ -38,7 +40,7 @@ private class Tokens(val context: Context) {
   val bpkSpacingLg = context.resources.getDimensionPixelSize(R.dimen.bpkSpacingLg)
   val bpkSpacingMd = context.resources.getDimensionPixelSize(R.dimen.bpkSpacingMd)
   val bpkSpacingSm = context.resources.getDimensionPixelSize(R.dimen.bpkSpacingSm)
-  val bpkBorderSizeLg = context.resources.getDimensionPixelSize(R.dimen.bpkBorderSizeLg)
+  val bpkBorderSizeLg = context.resources.getDimensionPixelSize(R.dimen.bpkBorderSizeSm)
```
Comment (tiagoengel) [inline Backpack/src/main/java/net/skyscanner/backpack/button/BpkButton.kt]: This doesn't look right, why is the name of the variable `borderSizeLg` and the token `borderSizeSm`?

File: Backpack/src/main/java/net/skyscanner/backpack/button/BpkButton.kt [hunk 2]
```diff
@@ -261,6 +266,23 @@ open class BpkButton : AppCompatButton {
         null
       )
     }
+    if (isElevated) {
```
Comment (tiagoengel) [inline Backpack/src/main/java/net/skyscanner/backpack/button/BpkButton.kt]: Is the elevation never added?

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:8413d2d287025e62b32bae8aa81b85169eb2e61a`

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:87a895505d52d3dc4c568e7428349666d8c6b115`

**Pull request merged**